### PR TITLE
ci: add pr label check and auto pr labeler

### PR DESCRIPTION
**Pull request by DeepakBomjan**

### CODE REVIEW

File: .github/workflows/check-pr-label.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: PR Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
```
Comment (debendraoli) [inline .github/workflows/check-pr-label.yaml:9]: Typo

File: .github/workflows/check-pr-label.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: PR Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
```
Comment (debendraoli) [inline .github/workflows/check-pr-label.yaml:6]: Perhaps past tense?

File: .github/workflows/auto-pr-labeler.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+name: Pull Request Labeler
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
```
Comment (debendraoli) [inline .github/workflows/auto-pr-labeler.yaml:7]: Past tense?

Comment (DeepakBomjan) [reply .github/workflows/auto-pr-labeler.yaml:7]: The trigger activities are correct as mentioned in [github doc](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

**Review by debendraoli — APPROVED** [review]
LGTM

**Review by izyak — APPROVED** [review]

**Pull request merged**